content/en/docs/release-notes/release-notes-1.4.md

```diff
@@ -8,21 +8,19 @@ type: "docs"
 Special thanks to the external contributors who contributed to this release:
 
 * [@andreas-p](https://github.com/andreas-p)
+* [@erikgb](https://github.com/erikgb)
+* [@eddiehoffman](https://github.com/eddiehoffman)
+* [@inteon](https://github.com/inteon)
+* [@anton-johansson](https://github.com/anton-johansson)
+* [@edglynes](https://github.com/edglynes)
+* [@jandersen-plaid](https://github.com/jandersen-plaid)
+* [@foosinn](https://github.com/foosinn)
+* [@jsoref](https://github.com/jsoref)
+* [@clatour](https://github.com/clatour)
+* [@tamalsaha](https://github.com/tamalsaha)
 
 # Deprecated Features and Breaking Changes
 
-## Certificate renewal period calculation
-
-There have been slight changes in the renewal period calculation for
-certificates. Renewal time (`rt`) is now calculated using formula `rt = notAfter -
-rb` where `rb = min(renewBefore, cert duration / 3)`. (See [docs](../../usage/certificate/#renewal) for more
-detailed explanation). Previously this was calculated using formula `rt =
-notAfter - rb`  where  `if cert duration < renewBefore; then rb = cert duration
-/ 3; else rb = renewBefore`. This change fixes a bug where, if a certificate's
-duration is very slightly larger than `renewBefore` period, then a cert gets
-renewed at `notAfter - renewBefore` which can lead to a continuous renewal loop,
-see [`cert-manager#3897`](https://github.com/jetstack/cert-manager/issues/3897).
-
```

Comment:
I moved this to the "Bug Fixes" section (first entry) since I think it is now a breaking change anymore... or is it still?

@irbekrm

Reply:
It does change the renewal behaviour if your certs have a duration of more than 90 days and `renewBefore` has not been set. Previously in that case the certs would have been renewed 30 days before expiry whereas now it will be 2/3 through the duration. Do you think that's worth being added as a change?

Reply:
Yes, I think that's a notable (and maybe breaking) change.

I moved the bug fix from under "Bug Fixes" to "Breaking Changes" in 119faf9 with a way to keep the old behavior by setting renewBefore to `720h`.

What do you think?

Reply:
This looks great to me!